packages/core/src/Kpi/Kpi.stories.tsx

```diff
@@ -64,6 +59,10 @@ export const Main: StoryObj<HvKpiProps> = {
     },
     classes: { control: { disable: true } },
   },
+  parameters: {
+    // Enables Chromatic snapshot
+    chromatic: { disableSnapshot: false },
```

Comment:
Isn't a delay needed given the previous waitBeforeCapture?

Reply:
Looking at the image taken by Chromatic it does not seem needed:
![image](https://github.com/lumada-design/hv-uikit-react/assets/14977550/a33b2c99-db68-487a-b54c-618ce4d990fc)